test/operator-framework-e2e/generate_dockerfile.go

```diff
@@ -0,0 +1,36 @@
+package operatore2e
+
+import (
+	"os"
+	"path/filepath"
+	"text/template"
+)
+
+// generates Dockerfile and its contents for a given set of yaml files
+func generateDockerFile(dockerFilePath, yamlFolderName, dockerFileName string) error {
+	t, err := template.New("dockerfile").Parse(dockerfileTmpl)
+	if err != nil {
+		panic(err)
+	}
+
+	dockerFilePath = filepath.Join(dockerFilePath, dockerFileName)
+	file, err := os.Create(dockerFilePath)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	_, err = file.WriteString("FROM scratch\n")
+	if err != nil {
+		return err
+	}
+	err = t.Execute(file, struct{ YamlDir string }{yamlFolderName})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+const dockerfileTmpl = `ADD {{.YamlDir}} /configs/{{.YamlDir}}
+`
```

Comment:
Just making sure you want a blank line here? Perhaps make it more obvious by using `\n` instead? Because, this could be taken for a typo (and use double-quotes instead of back-ticks)